front/src/interfaces.ts

```diff
@@ -73,4 +73,15 @@ export interface ApplicantResponse {
 export interface Comment {
   commenter_name: string;
   content: string;
+}
+
+export interface ApplicantRow {
+  id: Application["id"];
```

Comment:
This is an interesting way of defining the type, did you read up on this somewhere or just used it like that before?

Reply:
since Rows are extracted from Applications (ApplicantsPage.svelte, line 69), I thought to reuse/reference Application directly instead of copy/pasting over types. the reuse seems good but idk if it is following protocol, what do you think?

Reply:
You are bringing a good point about reusability, but I would prefer it to be done a little bit differently. I would create `ApplicationBase` interface, which would be the base of the `Application` interface, essentially sharing all the same fields between `Application` and `ApplicationRow`. Then `Application` and `ApplicationRow` would extend `ApplicationBase` (more on extending here https://www.typescripttutorial.net/typescript-tutorial/typescript-extend-interface/). In your case, `ApplicationBase` will be you `ApplicationRow`, but for clarity you can just have empty `ApplicationRow` interfaces that extends `ApplicationBase`